extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeGroup.php

```diff
@@ -42,13 +58,21 @@ public function getSource() {
 		return $this->getSourceForChildren();
 	}
 
-	/**
-	 * @return string
-	 */
+	protected function showIncomplete() {
+		return strcmp( $this->getDisplay(), self::SHOW_INCOMPLETE_OPTION ) === 0;
```

Comment:
Per offline discussion, please make sure that case sensitivity is handles as you want it to be handled